cp3pt0-deployment/setup_tenant.sh

```diff
@@ -202,6 +211,7 @@ function install_nss() {
     fi
 
     wait_for_operator "$OPERATOR_NS" "ibm-namespace-scope-operator"
+    accept_license "namespacescope" "$OPERATOR_NS"
```

Comment:
need to move this to after configure_nss_kind because the NSS CR isn't created until then